data/campaigns/tutorial02_warfare.wmf/scripting/texts.lua

```diff
@@ -126,6 +126,37 @@ trainingcamp2 = {
    )
 }
 
+scouting1 = {
+   position = "topright",
+   title = _"Scout's Hut",
+   body = (
+      h1(_"Explore Your Surroundings") ..
+      p(_[[Large part of the map is not explored yet. It might be a good idea to know which direction is best for expanding our empire.]]) ..
+      p(_[[You can try your luck and expand randomly, but it is risky. Safe way is to use scout to explore unseen parts of the map. What if he finds mountains overflowing with gold?]]) ..
+      p(_[[We will need a scout hut to start an expedition. It is a small building.]]) ..
```

Comment:
```suggestion
      p(_[[We will need a scout’s hut to start exploring. It is a small building.]]) ..
```
We already use "expedition" for seafaring, so I'd rather not confuse the terminology.